packages/react-instantsearch-core-next/rollup.config.js

```diff
@@ -0,0 +1,74 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import filesize from 'rollup-plugin-filesize';
+import globals from 'rollup-plugin-node-globals';
+import resolve from 'rollup-plugin-node-resolve';
+import replace from 'rollup-plugin-replace';
+import { uglify } from 'rollup-plugin-uglify';
+
+const clear = x => x.filter(Boolean);
+
+const version = process.env.VERSION || 'UNRELEASED';
+const algolia = '© Algolia, inc.';
+const link = 'https://github.com/algolia/react-instantsearch';
+const createLicence = name =>
+  `/*! React InstantSearch${name} ${version} | ${algolia} | ${link} */`;
+
+const plugins = [
+  babel({
+    exclude: ['../../node_modules/**', 'node_modules/**'],
+    extensions: ['.js', '.ts', '.tsx'],
+    rootMode: 'upward',
+    runtimeHelpers: true,
+  }),
+  resolve({
+    browser: true,
+    preferBuiltins: false,
+    extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json'],
+  }),
+  commonjs(),
+  globals(),
+  replace({
+    'process.env.NODE_ENV': JSON.stringify('production'),
```

Comment:
What does `JSON.stringify` do here? Add quotes?

Reply:
Yes, to then strip it with Uglify.